src/arm-template/minimal/stock-service-template.json

```diff
@@ -121,6 +121,28 @@
     "stockServiceAppInsights": "stock-backend",
     "location": "[if(equals(parameters('location'), ''), resourceGroup().location, parameters('location'))]",
     "stockServiceCosmosDbName": "[concat(parameters('prefix'), parameters('delimiter'), variables('stockServiceGroupName'), parameters('suffix'))]",
+    "stockServiceCosmosDbDatabases": {
+      "transaction": {
+        "name": "StockBackend",
+        "throughput": 400,
+        "collections": [
+          {
+            "name": "StockTransaction",
+            "partitionKeyPath": "/terminalCode"
+          }
+        ]
+      },
+      "lease": {
+        "name": "StockLease",
+        "collections": [
+          {
+            "name": "leases",
+            "throughput": 400,
```

Comment:
@dzeyelid It is not necessary to specify RU at the collection level.

Reply:
I have not confirmed the environment construction, but I think it is OK.

How about creating 'leases' collection in 'StockBackend' DB without creating 'StockLease' DB in the sense of reducing resource costs?
But this may be the wrong architecture...